diff --git a/other/freetype/freetype.lua b/other/freetype/freetype.lua
index 3c7f918..879cb0e 100644
--- a/other/freetype/freetype.lua
+++ b/other/freetype/freetype.lua
@@ -23,14 +23,13 @@ FreeType = {
 		end
 		
 		local apply = function(option, settings)
-			-- include path
-			settings.cc.includes:Add(FreeType.basepath .. "/include")
 			
 			if option.use_pkgconfig == true then
 				settings.cc.flags:Add("`pkg-config freetype2 --cflags`")
 				settings.link.flags:Add("`pkg-config freetype2 --libs`")
 				
 			elseif option.use_winlib > 0 then
+				settings.cc.includes:Add(FreeType.basepath .. "/include")
 				if option.use_winlib == 32 then
 					settings.link.libpath:Add(FreeType.basepath .. "/lib32")
 				else
